inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseService.java

```diff
@@ -355,4 +355,12 @@ RepositoryImplConfig getKnowledgeBaseConfig(KnowledgeBase kb)
     List<KBQualifier> listQualifiers(KnowledgeBase kb, KBStatement aStatement);
 
     boolean statementsMatchSPO(KnowledgeBase akb, KBStatement mockStatement);
+    
+    /**
+     * Define base properties of comment, label and subClassOf while initializing the KB
+     * @param kb The knowledge base to initialize base properties
+     * @param kbUpdateFlag Flag variable to indicate the update of KB or not.  
+     */
+    void defineBaseProperties(KnowledgeBase kb, boolean kbUpdateFlag);
```

Comment:
Why do you pass the `kbUpdateFlag`? `KnowledgeBase` has a field `readOnly`

Reply:
Its a new feature in there , where we create a default list of properties and when we update the set of schema values in there, we need this flag.